scripts/ngi_pipeline_start.py

```diff
@@ -6,13 +6,32 @@
 cluster (UPPMAX for NGI), or trigger analysis itself.
 """
 import argparse
+import os
 
 from ngi_pipeline.conductor import flowcell
+from ngi_pipeline.conductor import launchers
 from ngi_pipeline.log.loggers import minimal_logger
 from ngi_pipeline.server import main as server_main
+from ngi_pipeline.utils.filesystem import recreate_project_from_filesystem
 
 LOG = minimal_logger("ngi_pipeline_start")
 
+
+class ArgumentParserWithTheFlagsThatIWant(argparse.ArgumentParser):
```

Comment:
It was either this or one of my other ideas, `ArgParseBaseClass` or `ArgParseSubclass`, but it's more of an in-between class so I went with the most descriptive name.